src/frontends/paddle/src/proto/framework.proto

```diff
@@ -15,8 +15,6 @@ limitations under the License. */
 syntax = "proto2";
 package paddle.framework.proto;
 
-option optimize_for = LITE_RUNTIME; // Added by Intel Corporation 2021-2022
```

Comment:
why is it removed? 
It's required for binary size optimizations

Reply:
Sorry, I forgot to add it when updating the proto file. @AndSonder , can you help add it and have a try whther it can work?

Reply:
> Sorry, I forgot to add it when updating the proto file. @AndSonder , can you help add it and have a try whther it can work?

added.

Reply:
please remove the debug log cause the issue is solved.